### PR TITLE
feat: Setting the default task list

### PR DESCRIPTION
**Pull request by nickrandolph**
GitHub Issue: #132 

Navigating to Important list, or the last list that the user had opened

## PR Type

What kind of change does this PR introduce?
- Feature
- Bugfix

## What is the current behavior?

- App opens with no task list selected

- Mock files don't load on all platforms
- Mock disabled by default in debug
- Android can't navigate to tasklist

## What is the new behavior?

- App opens the Important task list, or the task list that the user previously had opened

- Mock files now load (path is corrected to be case correct)
- Mock is enabled by default when in DEBUG
- Android (narrow screens) navigation works (except there are other issues with tasklistpage at the moment)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/ToDo.UI/Views/HomePage.xaml [hunk 1]
```diff
@@ -63,7 +67,9 @@
 							 IsSettingsVisible="False"
 							 Style="{StaticResource HomeNavigationViewStyle}"
 							 CompactModeThresholdWidth="0"
-							 ExpandedModeThresholdWidth="0">
+							 ExpandedModeThresholdWidth="0"
+							 SelectedItem="{Binding SelectedList.Value}"
+							 SelectionChanged="SelectedListChanged">
```
Comment (nickrandolph) [inline src/ToDo.UI/Views/HomePage.xaml:72]: @dr1rrb Is there a way we can bind events to commands? perhaps using a behaviour?

File: src/ToDo.UI/App.xaml.host.cs [hunk 2]
```diff
@@ -1,3 +1,7 @@
+#if DEBUG
+#define USE_MOCKS
+#endif
+
```
Comment (carldebilly) [inline src/ToDo.UI/App.xaml.host.cs:4]: This is not required anymore: it's not defined in the csproj.

**Review by carldebilly — APPROVED** [review]

File: src/ToDo.UI/App.xaml.host.cs (hunk 2 above)
Comment (nickrandolph) [reply src/ToDo.UI/App.xaml.host.cs:4]: This should be left in for the moment - it's really annoying to work on the application without having mocks enabled, so they should be no by default in debug imho. Once my other PR goes through to push mocks to the config, then we can specify it in the appsettings.development.json which would be the best way

File: src/ToDo.UI/Views/HomePage.xaml (hunk 1 above)
Comment (Xiaoy312) [reply src/ToDo.UI/Views/HomePage.xaml:72]: Behavior or listen for changes on the data-bound value

**Review by agneszitte-nventive — APPROVED** [review]

**Pull request merged**